tests/AiDotNet.Tensors.Benchmarks/BaselineRunners/py/run_direct_ptx_vision_box_iou_competitors.py

```
@@ -0,0 +1,184 @@
+import torchvision
+from torchvision.ops import box_iou as torchvision_box_iou
+
+WARMUPS = 30
+SAMPLES = 101
+LAUNCHES = 25
+SHAPES = (("n256-m256", 256, 256), ("n1024-m256", 1024, 256),
+          ("n1024-m1024", 1024, 1024), ("n4096-m256", 4096, 256))
+
+
+def require_no_foreign_compute(label):
+    monitor = subprocess.run(["nvidia-smi", "pmon", "-c", "1", "-s", "u"],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)